docs/2021-ga-release-blog-post.md

```diff
@@ -0,0 +1,60 @@
+# Introducing the new Azure SDK Management Libraries for JS
```

Comment:
recommend using the full name "JavaScript" instead of JS

Reply:
I concur.  We should also update any instance of Javascript to JavaScript.

Reply:
Changed it